includes/discovery/libvirt-vminfo.inc.php

```diff
@@ -9,18 +9,23 @@
 
     $ssh_ok = 0;
 
+    $userHostname = $device['hostname'];
+    if (!is_null($config['libvirt_username'])) {
```

Comment:
I think isset or !empty would be prefered here.
https://www.virendrachandak.com/techtalk/php-isset-vs-empty-vs-is_null/